http/src/main/resources/reference.conf

```diff
@@ -189,7 +189,7 @@ gke {
     ]
     # See https://cloud.google.com/kubernetes-engine/docs/release-notes
     # As of 2020-10-09, 1.16.x is the default but it does not work with the Galaxy chart.
-    version = "1.15.12-gke.4000"
+    version = "1.16.15-gke.6000"
```

Comment:
Seeing if the default GKE version works 
![image](https://user-images.githubusercontent.com/5368863/104654419-0d16b680-568a-11eb-9eab-128c0a1586f7.png)